src/components/form-helpers/form-helpers.js

```diff
@@ -40,7 +40,7 @@ export const PasswordFieldItem = ({ className, fieldName, label, placeholder })
   return (
     <FieldItem className={concatClasses('field-item', className)} fieldName={fieldName}>
       <label htmlFor={fieldName}>
-        {label}{' '}
+        {label}
```

Comment:
Oh, take care of this, probably the space is missing now.

Reply:
why need this space? the eye icon is in the right

Reply:
Oh, maybe the space was not necessary. Before there was a word before the icon, but the word is not there anymore.